lib/utils/request.js

```diff
@@ -193,6 +192,72 @@ function MULTIGET (fhurls, where, cb) {
   });
 }
 
+/**
+ * Utility function to download files to a file path from a url
+ * @param params
+ * @param cb
+ */
+function downloadFile(params, cb){
+  var url = params.url;
+  var filePath = params.output;
+  var fileStream = fs.createWriteStream(filePath);
```

Comment:
The error should propagate up the pipe. Otherwise, we get the risk of doing a double callback.